docs/HubiC.md

```diff
@@ -25,13 +25,15 @@ Go to https://hubic.com/home/browser/developers/ and add an application. Applica
 name must be unique across hubiC, you can run the `hubic-application` command to have
 a unique application name suggested.
 
+
```

Comment:
Revert extra lines in this file to keep the diff clear. Unecessary as well since formated by markdown.